specification/security/resource-manager/readme.md

```diff
@@ -78,7 +78,7 @@ These are the global settings for the Security API.
 title: SecurityCenter
 description: API spec for Microsoft.Security (Azure Security Center) resource provider
 openapi-type: arm
-tag: package-composite-v3
+tag: package-preview-2023-05
```

Comment:
If this package-preview-2023-05 doesn't cover all the resources in composite-v3 tag, I will suggest to not change the default tag.